### ISSUE TITLE
questions about the DE Database

### ISSUE THREAD

**Issue by ShiZhongming** [architecture-building-systems/CityEnergyAnalyst#3573]
Hi @ddceruti 

Thank you again for leading the German (DE) Database.
I am opening this issue to check with you about two points in the DE database.

1. Under `assmblies/supply.xlsx, Tabs Heating and Hot_water`, multiple new entries of technologies have been added, however, they are not linked under `archetypes/construction_standard.xlsx, Tab Supply`. Is this intended? If not, do you think it is possible to update them accordingly?
2. Under `assmblies/supply.xlsx, Tabs Heating, BO4,5,6,7` are added yet defined under `components/conversion.xlsx, Tab Boiler`. Could define `BO4, 5, 6, 7` and add them to `components/conversion.xlsx, Tab Boiler`?

These two points are affecting Optimization script.

Thanks and regards, Zhongming

**Comment by ddceruti** [comment]
Hi,
I can update them, it will take a while because I will have to go through all TABULA standards manually and try to assign a modeled supply technology. I will try to get to it during this week.

**Comment by ShiZhongming** [comment]
Thank you! @ddceruti

**Comment by ddceruti** [comment]
Hi @ShiZhongming,
I have  two questions regarding the supply and conversion components. 

1) How is the total capex calculated? Because both have investment costs and different sizes available. For example:
Boiler section of conversion:
![grafik](https://github.com/architecture-building-systems/CityEnergyAnalyst/assets/52672045/9e4b46ca-d997-4794-9e97-cffe20b2a4b6)
Supply section of heating:
![grafik](https://github.com/architecture-building-systems/CityEnergyAnalyst/assets/52672045/bd1c48c3-219d-41ad-bb96-b55f33e1ec75)
What are I am supposed to define in these costs and efficiencies?

2) Can we define two conversion technologies for heating? I suppopse yes?
![grafik](https://github.com/architecture-building-systems/CityEnergyAnalyst/assets/52672045/8f9c40a6-aad0-4d4b-b6d0-e3d3bdfec3ed)


>     1. Under `assmblies/supply.xlsx, Tabs Heating and Hot_water`, multiple new entries of technologies have been added, however, they are not linked under `archetypes/construction_standard.xlsx, Tab Supply`. Is this intended? If not, do you think it is possible to update them accordingly? 
I have already implemented the heating and hot water supply technologies for the TABULA categories.

**Comment by MatNif** [comment]
Hi @ddceruti 

Slightly tricky questions to be honest. Let me try to formulate the best answer I can give you for now:

**1. Calculation of CAPEX**
So there are currently two spots in the databases where CAPEX is defined. One is the `SUPPLY` database and the other the `CONVERSION` database. The values indicated in the `SUPPLY` database are used by the `Life cycle analysis > System cost` script the formula indicated in the `CONVERSION` database is applied in all of `Optimization` scripts. I think that this differentiation is mainly due to legacy an not so much a conscious design choice.  (i.e. I think it would be valid to join the two approaches in the future). 
As for now though, I think we can use the parameters given in the `CONVERSION` database to determine a sensible value in the `SUPPLY` database. The CAPEX of each of the components based on their installed capacity and determined using the following function f(x) = a + b*x^c + d*ln(x) + e*x*ln(x). By assuming a typical range of capacities installed in heating/cooling systems at a building scale, we can therefore derive the average CAPEX of these systems per kW by linear regression (in the cases where f(x) is not already a linear function without offset).
![image](https://github.com/architecture-building-systems/CityEnergyAnalyst/assets/93093343/8b3518f9-3ca3-4f28-9749-0610bb2ec95f)
x: Capacity of the component in W.
f(x): regression-based CAPEX of the component in USD.

**2. Defining two conversion technologies for heating**
You can generally indicate multiple conversion technologies per system, but some restrictions apply to that: 
The system technologies specified in the `SUPPLY` database are used to describe the base heating/cooling system for stand-alone buildings in the optimisation script (i.e. the buildings that are not connected to a district heating/cooling network). The system technologies that can be adequately modelled in the 'Optimization' scripts are generally limited to the ones mentioned in the settings of the "Supply system part II (centralised)" script for the moment though.
So in short, all of the solar technologies as well as fuel cells and storage technologies can currently not be indicated as a part of a system. So the specific system you would like to represent can not be adequately taken into consideration at the moment.
